### PR TITLE
Cloud.gov to USWDS 2 post

### PR DESCRIPTION
**Pull request by Dahianna**
Fixes issue(s) # .

[CircleCI](https://circleci.com/gh/18F/18f.gsa.gov/tree/uswds2-cloudgov)

[:sunglasses: PREVIEW](https://federalist-proxy.app.cloud.gov/preview/18f/18f.gsa.gov/uswds2-cloudgov/)

[Preview README for this branch](https://github.com/18F/18f.gsa.gov/blob/uswds2-cloudgov/README.md)

Changes proposed in this pull request:
- Publish post
- Add images

### CODE REVIEW

**Comment by lauraGgit** [comment]
I'm seeing a line on the right of the image and I'm not seeing the css images- which i can add to it in a bit.

**Comment by Dahianna** [comment]
I was not sure if you were creating the table as an image, so I didn't upload the css images.

File: _posts/2020-04-21-a-token-of-our-affection-uswds-2.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+---
+title: "A token of our affection - A field guide USWDS 2"
```
Comment (cmajel) [inline _posts/2020-04-21-a-token-of-our-affection-uswds-2.md:2]: Whoops, I see there was a typo / inconsistency in the title. Can we update this to: 
`A token of our affection - A field guide to USWDS 2`

Comment (Dahianna) [reply _posts/2020-04-21-a-token-of-our-affection-uswds-2.md]: Done!

File: _posts/2020-04-21-a-token-of-our-affection-uswds-2.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+- christine-bath
+- laura-gerhardt
+tags:
+- web design system
+- lessons learned
+- cloud.gov
+excerpt: "We recently worked with the cloud.gov team to update their public site, cloud.gov, to United States Web Design System 2. The USWDS provided concepts we were able to use to translate designs into code a lot faster and deliver higher fidelity results once we understood how to use them."
+---
+
+We recently worked with the cloud.gov team to update their public site, [cloud.gov](https://cloud.gov/), to United States Web Design System 2 (USWDS 2). This was our first time digging into Version 2 of the system.
```
Comment (cmajel) [inline _posts/2020-04-21-a-token-of-our-affection-uswds-2.md:14]: I realize we don't have a link to USWDS. Can update this line to add one? Suggestion below:

`We recently worked with the cloud.gov team to update their public site, [cloud.gov](https://cloud.gov/), to [United States Web Design System 2 (USWDS 2)](https://designsystem.digital.gov/).`

Comment (Dahianna) [reply _posts/2020-04-21-a-token-of-our-affection-uswds-2.md]: Added

File: _posts/2020-04-21-a-token-of-our-affection-uswds-2.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+- laura-gerhardt
+tags:
+- web design system
+- lessons learned
+- cloud.gov
+excerpt: "We recently worked with the cloud.gov team to update their public site, cloud.gov, to United States Web Design System 2. The USWDS provided concepts we were able to use to translate designs into code a lot faster and deliver higher fidelity results once we understood how to use them."
+---
+
+We recently worked with the cloud.gov team to update their public site, [cloud.gov](https://cloud.gov/), to [United States Web Design System 2 (USWDS 2)](https://designsystem.digital.gov/). This was our first time digging into Version 2 of the system.
+Using the latest version of the web design system lets us easily correct some usability concerns with the site, and work towards compliance with the [21st Century Idea Act web design standards](https://digital.gov/resources/21st-century-integrated-digital-experience-act/).
```
Comment (lauraGgit) [inline _posts/2020-04-21-a-token-of-our-affection-uswds-2.md:15]: should this be `let us`?

File: _posts/2020-04-21-a-token-of-our-affection-uswds-2.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+<figure>
+  <a href="{{site.baseurl}}/assets/blog/web-design-standards/cloud.gov-uswds-tokens.jpg">
+    <img src="{{site.baseurl}}/assets/blog/web-design-standards/cloud.gov-uswds-tokens.jpg" alt="A screenshot of a design file showing how multiple colors from cloud.gov's original color palette map to USWDS color tokens."/>
+  </a>
+  <figcaption>USWDS has a large color library, so we were able to find USWDS color tokens that closely matched existing brand colors.</figcaption>
+</figure>
+
+In the design files, we were able to use the USWDS color library to test
+and import colors into our design. USWDS color tokens include [a graded system](https://designsystem.digital.gov/design-tokens/color/overview/)
+that helps designers check color contrast on the fly, sowe could make
```
Comment (cmajel) [inline _posts/2020-04-21-a-token-of-our-affection-uswds-2.md:71]: Type `sowe` >> `so we`

File: _posts/2020-04-21-a-token-of-our-affection-uswds-2.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+Using the latest version of the web design system lets us easily correct some usability concerns with the site, and work towards compliance with the [21st Century Idea Act web design standards](https://digital.gov/resources/21st-century-integrated-digital-experience-act/).
+
+Designers and developers use different tools and vocabulary to deliver
+government digital services and the USWDS provided concepts we were able
+to use to translate designs into code a lot faster and deliver higher
+fidelity results once we understood how to use them.
+
+## Theming
+
+USWDS provides variables, scss, and structure for custom themes that lets teams customize their government websites. Themes allowed us to use the consistent development and design patterns of the USWDS to showcase cloud.gov’s unique brand identity. We learned that these themes are extensive and allow lots of flexibility to design sites that reflect an agency’s visual style. We leveraged the default user experience behavior of the USWDS 2, while updating the colors and typography to match the brand identity of cloud.gov.
```
Comment (lauraGgit) [inline _posts/2020-04-21-a-token-of-our-affection-uswds-2.md:24]: Would it make more sense to say "Themes allowed us to use the consistent development and design patterns of the USWDS _while_ showcasing cloud.gov’s unique brand identity."?

File: _posts/2020-04-21-a-token-of-our-affection-uswds-2.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+
+The themes are modified by updating the variables in [Sass (Syntactically Awesome Style Sheets)](https://en.wikipedia.org/wiki/Sass_(stylesheet_language))
+`dist/scss/theme` folder of the USWDS
+[project](https://designsystem.digital.gov/documentation/developers/#download).
+To update the theme, you’ll need a build system that recompiles your CSS. We used this built-in capability with Jekyll running on Federalist.
+For more on how the themes work, take [a look at the USWDS documentation](https://designsystem.digital.gov/documentation/developers/#customization-theming-and-tokens).
+The USWDS SCSS files are broken out by area including color, components,
+spacing, typography, and utilities. Since our theme included cloud.gov
+fonts, colors, and layouts we copied the entire theme directory and then
+referenced it from our main SCSS file,` index.scss`.
```
Comment (lauraGgit) [inline _posts/2020-04-21-a-token-of-our-affection-uswds-2.md:34]: change ` index.scss` to " `index.scss`.

File: _posts/2020-04-21-a-token-of-our-affection-uswds-2.md [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+To organize our custom styles, we followed an internal practice of
+placing new styles into our `index.scss` and put adapted USWDS element
+styles into the `uswds-theme-custom-styles.scss.` For applying themed
+elements we used the great [USWDS utility classes](https://designsystem.digital.gov/utilities/) if we wanted to
+change a single element, and the utility or token mixins if we wanted to
+apply that pattern to multiple elements or wanted to apply a number of
+styles to a single element or class. We may not have been perfect in
+keeping this consistent, but in general we tried to follow the principle
+of Don’t Repeat yourself (DRY).
+
```
Comment (lauraGgit) [inline _posts/2020-04-21-a-token-of-our-affection-uswds-2.md:45]: See my PR about adding back in the table.

File: _posts/2020-04-21-a-token-of-our-affection-uswds-2.md [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+## Colors
+
+We wanted cloud.gov to be easy to recognize , and color can be a
```
Comment (lauraGgit) [inline _posts/2020-04-21-a-token-of-our-affection-uswds-2.md:48]: there is an extra space after the work recognize

File: _posts/2020-04-21-a-token-of-our-affection-uswds-2.md [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+- laura-gerhardt
+tags:
+- web design system
+- lessons learned
+- cloud.gov
+excerpt: "We recently worked with the cloud.gov team to update their public site, cloud.gov, to United States Web Design System 2. The USWDS provided concepts we were able to use to translate designs into code a lot faster and deliver higher fidelity results once we understood how to use them."
+---
+
+We recently worked with the cloud.gov team to update their public site, [cloud.gov](https://cloud.gov/), to United States Web Design System 2 (USWDS 2). This was our first time digging into Version 2 of the system.
+Using the latest version of the web design system lets us easily correct some usability concerns with the site, and work towards compliance with the [21st Century Idea Act web design standards](https://digital.gov/resources/21st-century-integrated-digital-experience-act/).
```
Comment (cmajel) [inline _posts/2020-04-21-a-token-of-our-affection-uswds-2.md:15]: Looking at the digital.gov link, I think we should update `21st Century Idea Act web design standards` >> `21st Century IDEA Act web design standards`

File: _posts/2020-04-21-a-token-of-our-affection-uswds-2.md [hunk 10]
```diff
@@ -0,0 +1,148 @@
...
+<figure>
+  <a href="{{site.baseurl}}/assets/blog/web-design-standards/cloud.gov-uswds-tokens.jpg">
+    <img src="{{site.baseurl}}/assets/blog/web-design-standards/cloud.gov-uswds-tokens.jpg" alt="A screenshot of a design file showing how multiple colors from cloud.gov's original color palette map to USWDS color tokens."/>
+  </a>
+  <figcaption>USWDS has a large color library, so we were able to find USWDS color tokens that closely matched existing brand colors.</figcaption>
+</figure>
+
+In the design files, we were able to use the USWDS color library to test
+and import colors into our design. USWDS color tokens include [a graded system](https://designsystem.digital.gov/design-tokens/color/overview/)
+that helps designers check color contrast on the fly, sowe could make
```
Comment (lauraGgit) [inline _posts/2020-04-21-a-token-of-our-affection-uswds-2.md:71]: need to change sowe to "so we"

File: _posts/2020-04-21-a-token-of-our-affection-uswds-2.md [hunk 11]
```diff
@@ -0,0 +1,148 @@
...
+To update the theme, you’ll need a build system that recompiles your CSS. We used this built-in capability with Jekyll running on Federalist.
+For more on how the themes work, take [a look at the USWDS documentation](https://designsystem.digital.gov/documentation/developers/#customization-theming-and-tokens).
+The USWDS SCSS files are broken out by area including color, components,
+spacing, typography, and utilities. Since our theme included cloud.gov
+fonts, colors, and layouts we copied the entire theme directory and then
+referenced it from our main SCSS file,` index.scss`.
+
+To organize our custom styles, we followed an internal practice of
+placing new styles into our `index.scss` and put adapted USWDS element
+styles into the `uswds-theme-custom-styles.scss.` For applying themed
```
Comment (cmajel) [inline _posts/2020-04-21-a-token-of-our-affection-uswds-2.md:38]: Minor syntax fix

`uswds-theme-custom-styles.scss.` >> `uswds-theme-custom-styles.scss`

File: _posts/2020-04-21-a-token-of-our-affection-uswds-2.md [hunk 12]
```diff
@@ -0,0 +1,148 @@
...
+and map those colors to the USWDS2 theme using color tokens.
+
+USWDS2 has a [default color theme of blues and reds](https://designsystem.digital.gov/design-tokens/color/theme-tokens/),
+but also recognizes that not every government site should look the same.
+The system includes a [color library of 24 graded color families](https://designsystem.digital.gov/design-tokens/color/system-tokens/)
+that map to preset CSS variables and classes to give teams flexibility.
+This robust color system made it easy for us to find the USWDS color
+tokens that matched cloud.gov’s branding. It also gave us a vocabulary
+to talk about colors without having to use hex codes. Soon, we were
+talking about how the sidebar background should be `accent-warm` (USWDS color token `gray-warm-10`) rather than`#E6E6E2`.
```
Comment (cmajel) [inline _posts/2020-04-21-a-token-of-our-affection-uswds-2.md:60]: Syntax fix to add a space:

rather than`#E6E6E2` >> rather than `#E6E6E2`

**Comment by cmajel** [comment]
@Dahianna I added a few language / syntax fix comments.

I also see the line @lauraGgit is seeing and think it was an issue on my end when I added it to the post. I re-cropped it to remove it (see below)

---

cloud gov-uswds-tokens

**Pull request merged**